lib/matplotlib/tests/test_figure.py

```diff
@@ -1819,3 +1819,18 @@ def test_subfigure_stale_propagation():
     sfig2.stale = True
     assert sfig1.stale
     assert fig.stale
+
+
+@pytest.mark.parametrize("input, output", [
+    ((6, 4), (6, 4)),
+    ((6, 4, "in"), (6, 4)),
+    ((5.08, 2.54, "cm"), (2, 1)),
+    ((600, 400, "px"), (6, 4)),
+])
+def test__parse_figsize(input, output):
```

Comment:
I'm not a stickler for this, but don't we usually avoid unit tests for private methods, preferring to test the user facing functionality?   This could be easily done by making a figure and then checking its size in inches, which is basically what you are doing here anyway.